addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -153,7 +153,7 @@ def _l10n_in_edi_ewaybill_cancel_invoice(self, invoices):
             if "312" in error_codes:
                 # E-waybill is already canceled
                 # this happens when timeout from the Government portal but IRN is generated
-                error_message = "<br/>".join(["[%s] %s" % (e.get("code"), html_escape(e.get("message"))) for e in error])
+                error_message = "<br/>".join([html_escape("[%s] %s" % (e.get("code"), e.get("message"))) for e in error])
```

Comment:
Dont we want to also have explicit names ? same for the other htlm escape :smile:

Reply:
I don't think so. This is not getting translated and isn't error prone either so I'd keep it like that (imo)